README.md

```diff
@@ -1 +1,93 @@
-# Hello-Word
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+ 
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- 
+
+<!-- PROJECT DESCRIPTION -->
+# 📖  <a name="about-project">Hello Word</a>
+> In this project we wanna just great personne who will see our page  We call this project Greating We built this project just by using html and css
+Hello Word is an exercise 
+## 🛠 Built With <a name="built-with">Visual studio</a>
+### Tech Stack <a name="tech-stack"></a>
+> In this project we use just one page html and his document css to make it pretty
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">index.html</a></li>
+    <li><a href="#">styles.css</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+### Key Features <a name="key-features">Hello Microverse</a>
+> We have just one title :
+- Hello Microverse
+
```

Comment:
- [ ] You did well😃 but as stated by the previous reviewer 💯, kindly update the readme with clear information, so kindly here you can replace the key features with clear other features🪄 like : 
``` Add the HTML for the structure of the project and Add the heading hello microverse etc...```⤵️